legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/tests/store.pure

```diff
@@ -60,7 +63,7 @@ function meta::external::store::service::tests::store::TradeProductServiceStore(
       id     = 'AllTradeService',
       path   = '/allTradesService',
       method = HTTPMethod.GET,
-      parameters = ^ServiceParameter(name = 'serializationFormat', type = ^StringTypeReference(list = false), location = Location.QUERY),
+      parameters = ^ServiceParameter(name = 'serializationFormat', type = ^StringTypeReference(list = false), location = meta::external::store::service::metamodel::Location.QUERY),
```

Comment:
Do we need these changes to packages?

Reply:
We have a conflict here with Location coming from core_authentication. So we need full qualified name